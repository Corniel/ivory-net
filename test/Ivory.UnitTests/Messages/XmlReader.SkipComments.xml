<document>
  <!-- Should be removed -->
  <result>Should stay</result>
</document>
