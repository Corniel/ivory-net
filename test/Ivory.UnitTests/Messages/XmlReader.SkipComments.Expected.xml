<document>
  <result>Should stay</result>
</document>
